easybuild/easyconfigs/b/bcbio-gff/bcbio-gff-0.6.7-foss-2021a.eb

```diff
@@ -0,0 +1,42 @@
+# Contribution by
+# DeepThought, Flinders University
+# Updated to v0.6.7
+# R.QIAO <rob.qiao@flinders.edu.au>
+
+easyblock = 'PythonPackage'
+
+name = 'bcbio-gff'
+version = '0.6.7'
+
+homepage = 'https://github.com/chapmanb/bcbb/tree/master/gff'
+
+description = """
+Read and write Generic Feature Format (GFF) with Biopython integration.
+"""
+
+toolchain = {'name': 'foss', 'version': '2021a'}
+
+source_urls = [PYPI_SOURCE]
+sources = ['%(name)s-%(version)s.tar.gz']
+checksums = ['6e6f70639149612272a3b298a93ac50bba6f9ecece934f2a0ea86d4abde975da']
+
+dependencies = [
+    ('Python', '3.9.5'),
+    ('SciPy-bundle', '2021.05'),
+]
+
+download_dep_fail = True
+use_pip = True
+sanity_pip_check = True
+
+bcbiogffroot = 'lib/python%(pyshortver)s/site-packages'
```

Comment:
@robqiao as with the other version, please use the `local_` prefix for these variables cf. https://easybuild.readthedocs.io/en/latest/Easyconfig-files-local-variables.html